lib/athena_web/admin/controllers/location/index.html.heex

```diff
@@ -35,17 +42,16 @@
   </tbody>
 </table>
 
-<h2><%= gettext("actions") %></h2>
+<%!--
```

Comment:
@screeper I think sobelow is failing because of those comments.

We should delete them anyways before merging this PR.